common/web/Makefile.mk

```diff
@@ -40,6 +40,11 @@ common_web_libolaweb_la_SOURCES = \
     common/web/SchemaParser.cpp \
     common/web/SchemaParser.h
 
+if USING_WIN32
+#Work around limitations with Windows library linking
+common_web_libolaweb_la_LIBADD = common/libolacommon.la
```

Comment:
Should this actually be here all the time, it is for plugins, although I guess they aren't within it, or is libolaweb supposed to be linked into libolacommon and it's just the Windows limitations that break it?